packages/blocks/src/components/Filter/Tag.tsx

```diff
@@ -7,13 +7,15 @@ type TagProps = {
   onClose?: () => void;
   onClick?: () => void;
   children?: React.ReactNode;
+  closable?: boolean;
 };
 
 export const Tag: React.FC<TagProps> = ({
   onClose,
   onClick,
   children,
   skin,
+  closable
```

Comment:
I don't think we need `closable` props.
We can determine if onClose is existing then it's closable